pages/proposals/proposals.jsx

```diff
@@ -66,25 +66,60 @@ var Proposal = React.createClass({
 
   },
   formatProposal(proposal) {
+    const FIELD_OPTIONS = this.props.localizedStrings.form_field_options;
+    const SPACES = FIELD_OPTIONS.spaces;
+    const TIME = FIELD_OPTIONS.timeneeded;
+    const LANGUAGES = FIELD_OPTIONS.languages;
+
     let formatted = Object.assign({}, proposal);
+
+    // Although Space labels are localized, when we submit the entry to
+    // Google Spreadsheet and GitHub we want them to be in English.
+    // This is to make the curation process simpler.
+    for (let key in SPACES) {
+      if (SPACES[key] === formatted.space) {
+        formatted.space = EnglishStrings.form_field_options.spaces[key];
```

Comment:
given how much it's accessed, it's probably worth doing:
```
const ffOptions = EnglishStrings.form_field_options;
const language = ffOptions.languages
```
etc. This makes the code a little less long (line length wise) and keeps us from constantly referencing EnglishStrings.form_field_options